gradient/cli/notebooks.py

```diff
@@ -24,72 +24,83 @@ def notebook_metrics():
     pass
 
 
-# @notebooks_group.command("create", help="Create new notebook")
-# @click.option(
-#     "--vmTypeId",
-#     "vm_type_id",
-#     type=int,
-#     required=True,
-#     help="Type of Virtual Machine",
-#     cls=common.GradientOption,
-# )
+@notebooks_group.command("create", help="Create new notebook")
 @click.option(
-    "--containerId",
-    "container_id",
-    type=int,
+    "--clusterId",
+    "cluster_id",
+    type=str,
     required=True,
-    help="Container ID",
+    help="Cluster ID",
     cls=common.GradientOption,
 )
 @click.option(
-    "--clusterId",
-    "cluster_id",
+    "--vmTypeLabel",
```

Comment:
Could you change --vmTypeLabel to --machineType? The later is used in other places, like experiments and deployments.